### PR TITLE
added an example in proxy_cache_valid on system level inheritance; ad…

### PR DESCRIPTION
**Pull request by ronyayoung**
…ded near china info in zh faq file

### CODE REVIEW

File: zh/docs/edge-logic/faq.md [hunk 1]
```diff
@@ -98,14 +98,22 @@ proxy_ignore_headers Vary;
...
 
 如果您有一个或多个ICP备案域名并希望它们在中国大陆加速，请联系网宿（CDNetworks）技术支持提交关于您业务的所有必要信息。我们将这些信息确认无误后，CDN Pro 就将为您开启中国大陆节点的使用权限。然后您可以按以下步骤来启用这些中国大陆节点：
 
 1. 在该域名的 [加速项](</docs/portal/edge-configurations/creating-property.md>) 配置中，将“有ICP备案”设置成是。这样做可使加速项上的配置部署到中国大陆服务器，当这些服务器接收客户端请求时会正常响应文件。否则它们将返回状态代码 451。
 
 2. 创建携带“有ICP备案”为是的 [边缘域名](</docs/portal/traffic-management/creating-edge-hostname.md>) , 并将您的业务域名流量通过 CNAME 的方式解析到到此边缘域名上。这样GSLB调度系统就会将您的业务域名自动解析到上述1中描述的 CDN Pro 的中国大陆服务器上。
 
+#### 无ICP备案域名中国大陆加速 - Near China 专线解决方案
+
+如果您的CDN域名没有ICP备案，但仍然需要加速并分发内容到中国大陆，CDN Pro 为您提供了Near China专线的解决方案。 这个解决方案利用 CDN Pro 的特殊服务组nearChina，以部署在香港的优质节点线路，底延时高性能的下发您的内容到中国大陆。
```
Comment (nibinqtl) [inline zh/docs/edge-logic/faq.md:113]: Please change "下发" to "分发"。
Did you update the English version?

**Review by nibinqtl — CHANGES_REQUESTED** [review]

File: zh/docs/edge-logic/faq.md (hunk 1 above)
Comment (ronyayoung) [reply zh/docs/edge-logic/faq.md]: Ok, will change "下发" to "分发"。
Yes，I have added the English version in the pull of ronya-ngdoc-181.

File: en/docs/edge-logic/supported-directives.md [hunk 2]
```diff
@@ -722,7 +722,21 @@ Determines in which cases a stale cached response can be used during communicati
...
+
+The configuration at the server level is inherited by a location block only when this directive is not present in the location block. If you need the directive to configure on both server level and location block, use the directive as in the following example:
```
Comment (burtonquantil) [inline en/docs/edge-logic/supported-directives.md:727]: The first sentence is really confusing to me.  @nibinqtl  However, I'm not sure I understand it enough to suggest a rewording yet.

I suggest the second can be improved to:
If you need to use the proxy_cache_valid directive at both the server level and in a location block, use the following example as a guide:

**Review by burtonquantil — CHANGES_REQUESTED** [review]
I suggested a couple of changes.

**Comment by nibinqtl** [comment]
@ronyayoung please address Burton's suggestions.

**Review by burtonquantil — APPROVED** [review]
OK to merge

**Pull request merged**